packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -543,6 +545,7 @@ export default class BackupNg {
     removeJob: (id: string) => Promise<void>,
     worker: $Dict<any>,
   }
+
```

Comment:
Idem, even if that's annoying, I'll try to fix this on `master` instead :slightly_smiling_face: